scripts/track_file_sizes.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+gravityDir="packages/gravity-ui-web"
+distDir="$gravityDir/dist/ui-lib"
+cdnGravityDir="gravity-ui-web"
+historyFilename="$TRAVIS_BRANCH-file-size-history.js"
+historyFile="$gravityDir/$historyFilename"
+currentSizesFile="$gravityDir/currentfilesizes.json"
+reportPageName="filesizehistory.html"
+reportPage="$gravityDir/$reportPageName"
+
+if [ ! -z "$TRAVIS_COMMIT" ] && [[ "develop" == "$TRAVIS_BRANCH" || "master" == "$TRAVIS_BRANCH" ]]
```

Comment:
should we just do this on master?

would a build of either develop or master would overwrite the same file? or does it create separate files for each branch?

Reply:
each branch now and in the future have their own tracking files